lib/glimesh_web/templates/about/faq.html.md

```diff
@@ -2,161 +2,151 @@
+Our goal is to focus on having a live, diverse, and friendly platform where new users feel right at home after visiting your stream. We understand the need to continue to bring our user base back to the site, and make sure they know how they can find new content to watch. By showcasing all of our diverse streaming types directly in the nav bar, users can also immediately visit new types of streams they may be unaware exist.
 
+We also recognize that your growth as a streamer is our growth as a platform and we should do everything possible to make sure you are successful on Glimesh. We also recognize that your growth as a streamer is our growth as a platform and we should do everything possible to make sure you are successful on Glimesh. We want to ensure that we work closely with the community to help build the platform that you want to be a part of, and provide the tools you need to be successful.
 
 ##### Will Glimesh use my content in advertising? If yes, how so?
-The Terms of Service that binds your usage of our website gives us permission to use your content for advertising, redistribution, storage, and more. However we want to form strong bonds with streamers so appreciate us using their content to grow the platform and their stream.
 
+The Terms of Service that binds your usage of our website gives us permission to use your content for advertising, redistribution, storage, and more. However, we want to form strong bonds with streamers so appreciate us using their content to grow the platform and their stream.
 
-##### Will everyone have the opportunity to get a sub button? What are the requirements for a sub button?
-Yes! We’re offering subscriptions to all users in supported countries. The only requirement to set up your banking details is verifying your identity with our 3rd party and waiting for your bank to be connected. In addition, you are ultimately in control of your own stream so you can also disable your subscription button at will.
+##### Has everyone the opportunity to get a sub button? What are the requirements for a sub button?
 
-For initial alpha launch, we are targeting support for the US, Canada, and the UK.
+Yes! We’re offering subscriptions to all users in supported countries. The only requirement to set up your banking details is verifying your identity with our 3rd party and waiting for your bank to be connected. In addition, you are ultimately in control of your own stream so you can also disable your subscription button at will.
```

Comment:
with our 3rd party...payment processor?

what about W8 tax forms?

Reply:
good catch-
any suggestion on how to mention it properly?
maybe "...our 3rd party payment processor, waiting for your bank to be connected and after submitting your W8 BEN form, if you're not a resident of the USA."

Reply:
Probably good enough to say "our 3rd party payment & tax collection processors". We don't need to be specific on who, or for what.